gpiozero/boards.py

```diff
@@ -1053,6 +1053,71 @@ def __init__(self, pwm=False, initial_value=False, pin_factory=None):
         )
 
 
+class RGBLEDBoard(CompositeOutputDevice):
+    """
+    Extends :class:`CompositeOutputDevice`. and represents a generic RGBLED
+    board or collection of LEDs.
+
+    The following example turns on all the LEDs on a board containing 5 LEDs
+    attached to GPIO pins 2 through 6::
+
+        from gpiozero import RGBLEDBoard
+
+        leds = RGBLEDBoard((2, 3, 4), (5, 6, 7), (7, 8, 9))
+
+        leds.value = ((1, 0, 0), (0, 1, 0), (0, 0, 1))
+
+    Note that this class represents simple GPIO RGB LEDs (a GPIO pin per RGB
+    channel) as opposed to smarter LEDs such as neopixels.
+    """
+    def __init__(self, *args, **kwargs):
+        pwm = kwargs.pop('pwm', True)
+        active_high = kwargs.pop('active_high', True)
+        initial_value = kwargs.pop('initial_value', False)
+        pin_factory = kwargs.pop('pin_factory', None)
+        order = kwargs.pop('_order', None)
+        super(RGBLEDBoard, self).__init__(
+            *(
+                pins_or_board
+                if isinstance(pins_or_board, RGBLEDBoard) else
+                RGBLED(
+                    *pins_or_board, pwm=pwm, active_high=active_high, initial_value=(initial_value, ) * 3,
```

Comment:
Why have you got `initial_value=(initial_value, ) * 3,` ? IMHO RGBLEDBoard should have the same init-params as https://github.com/RPi-Distro/python-gpiozero/blob/master/gpiozero/output_devices.py#L581

Reply:
I grabbed some implementation from `LEDCollection` which uses `True` / `False` for `initial_value`. Why do `LEDCollection` and `LEDBoard` have boolean `initial_value` when `.value` itself is an n-tuple?

Reply:
I'm not sure - perhaps @waveform80 remembers more? 
You could argue that the `initial_value` for an `LEDCollection` should be a tuple-of-booleans of the same size as the `.value`, or you could argue that the `initial_value` is only a default value, so you'd want to initialise every LED in an LEDCollection with the same value? https://gpiozero.readthedocs.io/en/stable/api_boards.html#ledboard

I guess another option would be to for the `__init__` to support both an `initial_value` param (a boolean) **and** an `initial_values` param (a tuple of values), with an exception being raised if both are supplied, or an exception being raised if the number of tuples in `initial_values` is incorrect?

\*shrug\*

Oh, and is `LEDCollection` the class that allows nesting, i.e. each child-element can be an `LED` _or_ another `LEDCollection` object? I guess that would also make an `initial_values` param slightly more complex? (because you'd need to check that nested `LEDCollection`s have the same number of tuples / sub-nested `LEDCollection`s etc.)

Reply:
I think we need consistency foremost. So I'll go for `initial_value` being a single value for now. Do we have any classes where `initial_value` is a tuple the length (and shape) of the board itself? I do like the idea of an optional single value or tuple. The tricky thing (I think) is working out the length and shape at the point of looking it up in `kwargs`.

Reply:
Actually, on second thoughts, it makes more sense that it follows `RGBLED` in expecting a ringle 3-tuple (in this case all RGBLEDs would be set to the same colour). This is also consistent with `LEDBoard` in that `LEDBoard` takes the same `initial_value` as `LED` or `PWMLED`.

Anyone wanting to set a more specific range of colours can do so manually after init.

Reply:
> Actually, on second thoughts, it makes more sense that it follows RGBLED in expecting a ringle 3-tuple (in this case all RGBLEDs would be set to the same colour).

Haha, exactly as I suggested [above](https://github.com/RPi-Distro/python-gpiozero/pull/631#discussion_r177042139) ;-)

Reply:
Got there in the end!